src/chrome/content/rules/Conrad_Electronic.xml

```diff
@@ -0,0 +1,42 @@
+<!--
+	Invalid certificate:
+		produktinfo.conrad.com (expired)
+		www.produktinfo.conrad.com (expired, mismatch)
+		
+	Redirect to HTTP:
+		conrad.com
+		www.conrad.com
```

Comment:
On `conrad.com` you find a list of some more TLD used by this company.
It would be great if we can secure/comment all of them. See also our section about [enumerating subdomains](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains).
You may split the TLD into different files to avoid huge, complex rules that are hard to maintain. See `Amazon.xml` for an example.